website/templates/project/settings.mako

```diff
@@ -68,7 +68,27 @@
                   <span data-bind="css: messageClass, html: message"></span>
                   <span data-bind="if: disabled" class="help-block">
                     A top-level project's category cannot be changed
-                  </span>
+                  </span>               
+                  </p>
+                   
+                    <label for="title">Title</label>
+                    <input class="form-control" type="text" maxlength="200" data-bind="value: title, 
+                                                                            event: { keyup: dirtyTitleDescription }">
```

Comment:
Don't bind to an event - use Knockout's `computed` or `pureComputed`.

Knockout docs: http://knockoutjs.com/documentation/computedObservables.html